example/speedtest_example.py

```diff
@@ -0,0 +1,11 @@
+from python_mikrotik_login import MikrotikLogin
+
+username="admin"
+password="admin"
+url="http://10.4.2.1/login" #simulated Mikrotik login page url
```

Comment:
Url example simulasi yang lama jalan bro, g perlu diganti, tapi mungkin kasih keterangan:

`#this example url is working and you can run it to test the functionality or change it to your url`

ntar response dari urlnya w samain buat versi 3 ini

Reply:
yang lainnya udah oke

Reply:
oke noted

Reply:
bro kayaknya ada yang error di kode lu, beberapa udh w fix, tapi w ga ngerti dependensi yang lu pake dokumentasinya gimana. coba cek yg line 1:
`from typing import final`
w udah coba di linux & windows dua2nya g bisa diinstall

Reply:
sorry yang line 1 itu emang typo, kebawa auto suggest